crates/optimism/trie/src/proof.rs

```diff
@@ -0,0 +1,572 @@
+//! Provides proof operation implementations for [`OpProofsStorage`].
+
+use alloy_primitives::{
+    keccak256,
+    map::{B256Map, HashMap},
+    Address, Bytes, B256, U256,
+};
+use reth_db_api::DatabaseError;
+use reth_execution_errors::{StateProofError, StateRootError, StorageRootError, TrieWitnessError};
+use reth_primitives_traits::Account;
+use reth_trie::{
+    hashed_cursor::{
+        HashedCursor, HashedCursorFactory, HashedPostStateCursorFactory, HashedStorageCursor,
+    },
+    metrics::TrieRootMetrics,
+    proof::{Proof, StorageProof},
+    trie_cursor::{InMemoryTrieCursorFactory, TrieCursor, TrieCursorFactory},
+    updates::TrieUpdates,
+    witness::TrieWitness,
+    AccountProof, BranchNodeCompact, HashedPostState, HashedPostStateSorted, HashedStorage,
+    MultiProof, MultiProofTargets, Nibbles, StateRoot, StorageMultiProof, StorageRoot, TrieInput,
+};
+
```

Comment:
```suggestion
```
let's do the import preludes consistently without whitespace to match reth's preference